2014-06-01 11:08:01,265 - [INFO] - from play in pool-8-thread-5 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2014-06-01 11:08:11,087 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:08:13,700 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:11:34,930 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:11:35,072 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:20:06,790 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:20:06,934 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:20:07,798 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:20:07,940 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:20:30,298 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:20:30,416 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:31:16,060 - [ERROR] - from application in New I/O worker #5 


! @6id5acini - Internal server error, for (POST) [/removePost] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value postId]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2014-06-01 11:35:06,991 - [ERROR] - from application in New I/O worker #5 


! @6id5apme1 - Internal server error, for (POST) [/removePost] ->

play.PlayExceptions$CompilationException: Compilation error[value postId is not a member of Unit]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2014-06-01 11:35:19,423 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:35:19,538 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:35:19,622 - [ERROR] - from play in slick.db.execution.context-1 
Cannot invoke the action, eventually got an error: java.util.NoSuchElementException: None.get

2014-06-01 11:35:19,639 - [ERROR] - from application in slick.db.execution.context-1 


! @6id5b0f6d - Internal server error, for (POST) [/removePost] ->

play.api.Application$$anon$1: Execution exception[[NoSuchElementException: None.get]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at scala.Option.map(Option.scala:145) [scala-library-2.10.3.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library-2.10.3.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_05]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_05]
Caused by: java.util.NoSuchElementException: None.get
	at scala.None$.get(Option.scala:313) ~[scala-library-2.10.3.jar:na]
	at scala.None$.get(Option.scala:311) ~[scala-library-2.10.3.jar:na]
	at play.api.data.Form.get(Form.scala:218) ~[play_2.10-2.2.2.jar:2.2.2]
	at controllers.Application$$anonfun$2.apply(Application.scala:28) ~[classes/:na]
	at controllers.Application$$anonfun$2.apply(Application.scala:27) ~[classes/:na]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10$$anonfun$apply$11.apply(DBAction.scala:187) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10$$anonfun$apply$11.apply(DBAction.scala:186) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at scala.slick.backend.DatabaseComponent$DatabaseDef$class.withSession(DatabaseComponent.scala:31) ~[slick_2.10-2.0.0.jar:2.0.0]
	at scala.slick.jdbc.PlayDatabase.withSession(PlayDatabase.scala:6) ~[play-slick_2.10-0.6.0.1.jar:2.0.0]
	at play.api.db.slick.PredicatedDBAction$$anonfun$10.apply(DBAction.scala:164) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$10.apply(DBAction.scala:164) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10.apply(DBAction.scala:186) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10.apply(DBAction.scala:186) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library-2.10.3.jar:na]
	... 3 common frames omitted

2014-06-01 11:36:05,905 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:36:06,008 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:36:27,543 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:36:27,652 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:40:44,625 - [ERROR] - from application in New I/O worker #3 


! @6id5bj1pp - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[value id_= is not a member of models.database.Posts]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2014-06-01 11:42:24,816 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:42:24,994 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:44:11,229 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:44:11,336 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:46:41,124 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:46:41,218 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:47:14,228 - [ERROR] - from play in slick.db.execution.context-4 
Cannot invoke the action, eventually got an error: java.util.NoSuchElementException: None.get

2014-06-01 11:47:14,231 - [ERROR] - from application in slick.db.execution.context-4 


! @6id5cf68n - Internal server error, for (POST) [/compose] ->

play.api.Application$$anon$1: Execution exception[[NoSuchElementException: None.get]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at scala.Option.map(Option.scala:145) [scala-library-2.10.3.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:261) [play_2.10-2.2.2.jar:2.2.2]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$2.applyOrElse(PlayDefaultUpstreamHandler.scala:257) [play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library-2.10.3.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library-2.10.3.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:23) [scala-library-2.10.3.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_05]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_05]
Caused by: java.util.NoSuchElementException: None.get
	at scala.None$.get(Option.scala:313) ~[scala-library-2.10.3.jar:na]
	at scala.None$.get(Option.scala:311) ~[scala-library-2.10.3.jar:na]
	at play.api.data.Form.get(Form.scala:218) ~[play_2.10-2.2.2.jar:2.2.2]
	at controllers.PostController$$anonfun$submit$1.apply(PostController.scala:49) ~[classes/:2.2.2]
	at controllers.PostController$$anonfun$submit$1.apply(PostController.scala:48) ~[classes/:2.2.2]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10$$anonfun$apply$11.apply(DBAction.scala:187) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10$$anonfun$apply$11.apply(DBAction.scala:186) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at scala.slick.backend.DatabaseComponent$DatabaseDef$class.withSession(DatabaseComponent.scala:31) ~[slick_2.10-2.0.0.jar:2.0.0]
	at scala.slick.jdbc.PlayDatabase.withSession(PlayDatabase.scala:6) ~[play-slick_2.10-0.6.0.1.jar:2.0.0]
	at play.api.db.slick.PredicatedDBAction$$anonfun$10.apply(DBAction.scala:164) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$10.apply(DBAction.scala:164) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10.apply(DBAction.scala:186) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at play.api.db.slick.PredicatedDBAction$$anonfun$applyForDB$1$$anonfun$apply$10.apply(DBAction.scala:186) ~[play-slick_2.10-0.6.0.1.jar:0.6.0.1]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library-2.10.3.jar:na]
	... 3 common frames omitted

2014-06-01 11:52:57,579 - [ERROR] - from application in New I/O worker #3 


! @6id5d8k6c - Internal server error, for (GET) [/c] ->

play.PlayExceptions$CompilationException: Compilation error[Unmatched closing brace '}' ignored here]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2014-06-01 11:53:01,573 - [ERROR] - from application in New I/O worker #3 


! @6id5d9044 - Internal server error, for (GET) [/] ->

play.PlayExceptions$CompilationException: Compilation error[Unmatched closing brace '}' ignored here]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.10.3.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10-2.2.2.jar:2.2.2]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10-2.2.2.jar:2.2.2]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.10.3.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.10.3.jar:na]

2014-06-01 11:53:48,817 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:53:48,919 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-01 11:54:31,879 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost/nu_wasis_playlog_db

2014-06-01 11:54:31,972 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

